### PR TITLE
Adding additional fields to org

### PR DESCRIPTION
**Pull request by sumesh-aot**
*Issue #:*
https://github.com/bcgov/entity/issues/7782

*Description of changes:*
- Adding additional fields to org
- Remove unwanted columns from all code tables

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-auth license (Apache 2.0).

### CODE REVIEW

**Review by karthik-aot — APPROVED** [review]

File: auth-api/migrations/versions/08e1d49dfa1a_business_type_and_size.py [hunk 1]
```diff
@@ -0,0 +1,372 @@
...
+                               sa.PrimaryKeyConstraint('code')
+                               )
+
+    op.bulk_insert(
+        bus_size,
+        [
+            {
+                "code": "0-1",
+                "description": "1 Employee",
+                "default": True
```
Comment (saravanpa-aot) [inline auth-api/migrations/versions/08e1d49dfa1a_business_type_and_size.py:40]: at some point , we have to rename the "default" ..its again a sql keyword

File: auth-api/migrations/versions/7f3d5d328fd5_remove_unwanted_code_table_cols.py [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.drop_constraint('affidavit_status_created_by_id_fkey', 'affidavit_statuses', type_='foreignkey')
+    op.drop_constraint('affidavit_status_modified_by_id_fkey', 'affidavit_statuses', type_='foreignkey')
+    op.drop_column('affidavit_statuses', 'modified')
+    op.drop_column('affidavit_statuses', 'created_by_id')
+    op.drop_column('affidavit_statuses', 'created')
+    op.drop_column('affidavit_statuses', 'modified_by_id')
+    op.alter_column('business_size_codes', 'default',
```
Comment (saravanpa-aot) [inline auth-api/migrations/versions/7f3d5d328fd5_remove_unwanted_code_table_cols.py:28]: what does this alter_column do?

**Review by saravanpa-aot — APPROVED** [review]

File: auth-api/migrations/versions/7f3d5d328fd5_remove_unwanted_code_table_cols.py (hunk 2 above)
Comment (sumesh-aot) [reply auth-api/migrations/versions/7f3d5d328fd5_remove_unwanted_code_table_cols.py]: removed it

Comment (sumesh-aot) [reply auth-api/migrations/versions/7f3d5d328fd5_remove_unwanted_code_table_cols.py]: removed

**Pull request merged**